android/engine/src/main/java/org/smartregister/fhircore/engine/appfeature/model/HealthModule.kt

```diff
@@ -23,5 +23,6 @@ enum class HealthModule {
   FAMILY,
   CHILD,
   FAMILY_PLANNING,
+  FAMILY_CARE_PLAN,
```

Comment:
It was to prevent collision between these 2, because of differences in key

For Group resource:
   ```
 {
      "id": "family",
      "filterType": "TOKEN",
      "key": "code",
      "valueType": "CODEABLECONCEPT",
      "valueCoding": {
        "system": "https://www.snomed.org",
        "code": "35359004"
      }
    },
```


For CarePlan resource:
```
{
      "id": "family_care_plan",
      "filterType": "TOKEN",
      "key": "_tag",
      "valueType": "CODING",
      "valueCoding": {
        "system": "https://www.snomed.org",
        "code": "35359004"
      }
    },
```

Reply:
Sure, we can make it as a constant